orienteer-core/src/main/java/org/orienteer/core/component/command/DeleteODocumentCommand.java

```diff
@@ -49,10 +36,11 @@ public DeleteODocumentCommand(OrienteerDataTable<ODocument, ?> table, IModel<OCl
 	@Override
 	protected void performMultiAction(AjaxRequestTarget target, List<ODocument> objects) {
 		super.performMultiAction(target, objects);
-		ODatabaseDocument db = getDatabase();
-		db.commit(true);
-		db.begin();
-		db.getMetadata().reload();
+		DBClosure.sudoConsumer(db -> {
+			db.commit(true);
+			db.begin();
```

Comment:
`begin` is needed because of previous commit, so it's not needed here as well.

Reply:
Removed in 8975397